@xen-orchestra/web/src/components/LogoTextOnly.vue

```diff
@@ -1,5 +1,10 @@
 <template>
-  <svg :class="{ short }" :viewBox="short ? '65 622 125 64' : '65 622 735 64'" xmlns="http://www.w3.org/2000/svg">
+  <svg
+    :class="{ short }"
+    :viewBox="short ? '65 622 125 64' : '65 622 735 64'"
+    class="logoTextOnly"
```

Comment:
By convention we use kebab-case for classes:
```suggestion
    class="logo-text-only"
```